src/DependencyInjection/Admin/TaggedAdminInterface.php

```diff
@@ -43,6 +43,13 @@
  *     - The first and third argument are automatically injected by the AddDependencyCallsCompilerPass.
  *     - The second one is used as a reference of the Admin in the Pool, with the `setAdminClasses` call.
  *
+ * @method void   setCode(string $code)
```

Comment:
You can remove the `public function __construct(string $code, string $class, string $controller, ...);` information here

Reply:
Indeed, done.